docs/topics/db/examples/proxy_models.txt

```diff
@@ -0,0 +1,96 @@
+===========================
+Proxy model as relationship
+===========================
+
+You can use a proxy model, instead of its concrete model, as ``ForeignKey``,
+``ManyToManyField``, or ``OneToOneField``.
+
+We will use ``Student`` as our main proxy model:
+
+.. code-block:: python
+
+    from django.contrib.auth import get_user_model
```

Comment:
I think sticking with the `Person` model is a slightly clearer example - not everyone uses `django.contrib.auth`, and of those who do, most don't use `get_user_model()` since it's not necessary outside of third party packages.